gulpfile.js

```diff
@@ -38,7 +40,7 @@ gulp.task('coveralls', function () {
 gulp.task('testCI', ['lint', 'style', 'cover'], function () {
   if (process.env.NODE_ENV !== 'test') {
     gulp.src(process.env.COVERAGE_DIR + '/coverage')
-      .pipe(plugins.clean());
+      .pipe(vinylPaths(del));
```

Comment:
Seems like it might be a bit simpler (only one additional module) to use https://www.npmjs.com/package/gulp-rimraf here